src/core/functions/internal_functions/web/InternalModuleWeb.ts

```diff
@@ -80,4 +82,31 @@ export class InternalModuleWeb extends InternalModule {
             }
         };
     }
+
+    make_request(): (
```

Comment:
Can you rename this function to be `generate_request` to match the other function names in this file?